Apps/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
-add_subdirectory(Playground)
+if(NOT ANDROID)
+    add_subdirectory(Playground)
+endif()
```

Comment:
This is because we don't build Android from CMake?

Reply:
we don't buidl android playground from cmake. we use gradle.

Reply:
okie!